docs/src/test/scala/docs/http/scaladsl/ServerSentEventsExampleSpec.scala

```diff
@@ -0,0 +1,44 @@
+package docs.http.scaladsl
+
+import akka.NotUsed
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.sse.ServerSentEvent
+import akka.http.scaladsl.server.{ Directives, Route, RoutingSpec }
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.scaladsl.Source
+import docs.CompileOnlySpec
+import java.time.LocalTime
+import java.time.format.DateTimeFormatter.ISO_LOCAL_TIME
+import scala.concurrent.duration.DurationInt
+
+final class ServerSentEventsExampleSpec extends RoutingSpec with Directives with CompileOnlySpec {
```

Comment:
IIRC, not adding a test case will lead to leaking ActorSystems with `RoutingSpec` because of ScalaTest. So, maybe just don't extends from `RoutingSpec` and add dummy implicit definitions where needed, instead.